web/src/activity_ui.js

```diff
@@ -158,7 +162,7 @@ export function narrow_for_user_id(opts) {
     const person = people.get_by_user_id(opts.user_id);
     const email = person.email;
 
-    narrow.by("dm", email, {trigger: "sidebar"});
+    narrow_by_email(email);
```

Comment:
We should expect it to always be set. If we wrote something, it'd be an `assert` -- one should prefer asserts rather than writing `if` branch for a case that is not possible. You can use `import assert from "minimalistic-assert";` -- we mostly do it in TS files since it does type narrowing for us, but we use it in JS in at least one place.

Reply:
This `narrow_by_email` extraction would be a great prep commit/PR BTW -- it's distracting when reading the main commit.

Reply:
Great idea, I'll make it a prep commit right now :)

Reply:
Noting here that I chose not to add the assert because it was making writing tests more annoying (I had to make sure it was initialized) and I'm running low on time for working today so I'm just gonna go without the assert for now.